modules/examiner/test/ExaminerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Examiner module automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ .
+    "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+
+/**
+ * Examiner module automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class ExaminerTest extends LorisIntegrationTest
+{
+    protected $url = 'http://localhost/main.php?test_name=examiner';
```

Comment:
This shouldn't hardcode the hostname. LorisIntegrationTest looks it up from the config. See other tests that Ted wrote.